raglweb/__init__.py

```diff
@@ -320,12 +322,11 @@ def player(profile_id):
             matchup_count += 1
         matches.append(opponent)
 
-    cfg = app.config
-
-    # The final time should not change if some matches get canceled, so we take into account here
-    end_time = cfg['START_TIME'] + (matchup_count + matchup_canceled) * cfg['MATCHUP_DELAY'] / cfg['MATCHUP_COUNT']
+    group_stage_end_time = cfg['START_TIME'] + timedelta(weeks=cfg['GROUP_STAGE_WEEKS'])
+    # Calculate the fraction of the group stage that's complete.
+    group_stage_completion = min(1, max((0, (date.today() - cfg['START_TIME'])) / (group_stage_end_time - cfg['START_TIME'])))
```

Comment:
Is this actually working? I'm seeing a tuple division

Reply:
No, good spot. I broke it when I added the min/max.